play-routes-compiler/src/main/scala/CommandLinePlayRoutesCompiler.scala

```diff
@@ -63,6 +66,12 @@ object CommandLinePlayRoutesCompiler {
     }.text("Whether the reverse router should be namespaced. Useful if you have many routers that use the same actions.")
   }
 
+  private def stripHeader(path: String): Unit = {
+    val lines = Source.fromFile(path).getLines
+    val sansHeader = lines.span(line => line.matches("//.*|\\s*"))._2
```

Comment:
Does this look at the start of the line or do we need to add a `^` at the beginning of this regex to make sure that the regex only matches lines that start with this?

Reply:
And do we need to add a `$` at the end of these to make sure we're looking at the full line?

Reply:
I believe `matches` only returns true if it's a perfect match, but I can add that just in case